### DISCUSSION TITLE
Ender 3 V2 BLT and sprite pro version

### DISCUSSION THREAD

**Discussion by dom2587** [mriscoc/Ender3V2S1#187] [Q&A] [answered]
HI, Which version of the firmware do i need for the for an Ender 3 V2 with a BLT and a sprite pro?

**Comment by mriscoc** [comment, marked as answer]
We don't have an official answer about the thermistor on the Sprite Pro, but some users reports successful with this configuration: 
https://github.com/mriscoc/Special_Configurations/releases/tag/AM3950T13

Download for an Ender3V2 with board 4.2.2: [Ender3V2-422BLT-AM3950-20220503.bin](https://github.com/mriscoc/Special_Configurations/releases/download/AM3950T13/Ender3V2-422BLT-AM3950-20220503.bin)

Download for an Ender3V2 with board 4.2.7: [Ender3V2-427BLT-UBL-AM3950-20220420.bin](https://github.com/mriscoc/Special_Configurations/releases/download/AM3950T13/Ender3V2-427BLT-UBL-AM3950-20220420.bin)

**Reply by dom2587** [reply]
Thank you very much :)

**Comment by xonita75** [comment]
Hi there, I´m new to all this and i think i´m not able to compile a firmware to use in my printer. I have a Ender 3 max 4.2.2 boar, with color display, cr touch and sprite extruder pro installed. I´d like to know if you are planning on release the professional firmware so i ca use it on my printer. Thank you in advance.  Regards.

**Reply by mriscoc** [reply]
If you have the 4.2.2 board and a DWIN or DACAI screen (V2 or S1 printers) you can use the firmware. Look for the Special versions: https://github.com/mriscoc/Special_Configurations/releases/tag/AM3950T13

**Reply by xonita75** [reply]
Thank you very much. I'll give it a try. Regards.

**Comment by RonJongbloed** [comment]
I just installed a ender sprite direct extruder with a cr touch, now the homing is off the probe is outside the bed, is there a solution
using Ender3V2-422BLT-AM3950 firmware

**Reply by mriscoc** [reply]
Check: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
